schemas/value_sources.json

```diff
@@ -173,5 +179,26 @@
         "$ref": "https://eq.ons.gov.uk/value_sources.json#/calculated_summary_value_source"
       }
     ]
+  },
+  "progress_value_source": {
+    "description": "The progression status of a section or a block. When resolved it returns the string COMPLETED or any status or null.",
```

Comment:
> returns the string COMPLETED or any status or null.
This doesn't make sense?. Maybe be explicit that it returns null when not on the path otherwise the status?